dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -260,36 +253,55 @@ protected void generateNodeConfig(String logDir) {
         int port = PortHelper.getRandomPort();
         String endpoint = addr + ":" + port;
         String clientPort = " -client-port=";
-        String locators = "-locators=";
-        String locatorHost = null;
+        String locators = " -locators=";
+        String locPortString = " -peer-discovery-port=";
+        String locatorsList = null;
         String dirPath = snappyTest.getLogDir();
         String nodeLogDir = null;
         String timeStatistics = " -enable-time-statistics=" + SnappyPrms.getTimeStatistics() + " -statistic-archive-file=";
```

Comment:
can we change SnappyPrms.getTimeStatistics() to provide   the exact string?? for example SnappyPrms.getTimeStatistics() can return either " -enable-time-statistics=" + SnappyPrms.getTimeStatistics() + " -statistic-archive-file="; or empty